src/SDDPoptimize.jl

```diff
@@ -504,3 +504,81 @@ function add_cuts_to_model!(model::SPModel, t::Int64, problem::JuMP.Model, V::Po
     end
 end
 
+
+"""
+Prune all polyhedral functions in input array.
+
+Parameters:
+- model (SPModel)
+- params (SDDPparameters)
+- V Vector{PolyhedralFunction}
+    Polyhedral functions where cuts will be removed
+
+"""
+function prune_cuts!(model::SPModel, params::SDDPparameters, V::Vector{PolyhedralFunction})
```

Comment:
I would prefer exact_prune_cuts! to specify that this is not an heuristic (other pruning method will be implemented and might be wrapped in a generic prune_cuts() function later on